.husky/pre-commit.js

```diff
@@ -180,14 +186,14 @@ function getChangedFileExtensions(_folder) {
 }
 
 /**
- * Execute linting command
+ * Execute commands based on file extensions
  * @param {FOLDER, FILE_EXTENSION[]}
  */
-function executeLinting(_folder, _changedFileExtensions) {
+function executeCommands(_folder, _changedFileExtensions, _command) {
     for (const fileExtension of _changedFileExtensions) {
         const path = FOLDER_PATH[_folder];
-        const cmd = FILE_EXTENSION_LINTING_COMMAND[fileExtension];
-        console.log(`${fileExtension} change found in ${path}, linting...`);
+        const cmd = _command[fileExtension];
```

Comment:
```suggestion
        const cmd = _command[fileExtension];
        if (!cmd) return;
```

Ran on local, if `cmd` is undefined this (in case of finding FILE_EXTENSION_DOCUMENTATION_UPDATING_COMMAND for Typescript changes) then commit is blocked